check_borg

```diff
@@ -1,7 +1,9 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
Yes and no. Could you please elaborate, why? I know there are slight differences between `sh` and `bash`. I usually write for `bash`, because it's available on >90% of the targeted systems and it just rules out the slight differences.

Reply:
Because bash is not in FreebSD base, so this change is for portability reasons, as sh is the lowest common denominator. In my experience most bash-specific features are not required.